daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -1657,6 +1686,9 @@ func (devices *DeviceSet) Shutdown() error {
 	var devs []*devInfo
 
 	devices.Lock()
+	// Save DeviceSet Metadata first
```

Comment:
If there's a specific reason this should be done first, could you describe the *reason* here (so that other people will not move it around)? Otherwise this comment is redundant